content/en/data_jobs/databricks.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Enable Data Jobs Monitoring for Databricks
+kind: documentation
+further_reading:
+    - link: '/data_jobs'
+      tag: 'Documentation'
+      text: 'Data Jobs Monitoring'
+---
+
+{{< callout url="https://forms.gle/PZUoEgtBsH6qM62MA" >}}
+Data Jobs Monitoring is in private beta. Fill out this form to join the wait list.
+{{< /callout >}} 
+
+[Data Jobs Monitoring][7] gives visibility into the performance and reliability of your Apache Spark and Databricks jobs.
+
+## Setup
+
+Follow these steps to enable Data Jobs Monitoring for Databricks.
+
+1. [Configure the Datadog-Databricks integration](#configure-the-datadog-databricks-integration) with your Databricks API token.
+1. [Install the Datadog Agent](#install-the-datadog-agent-on-your-databricks-clusters) on your Databricks cluster(s).
+1. [Add your Datadog API key](#add-your-datadog-api-key-in-databricks) in Databricks.
+
+### Configure the Datadog-Databricks integration 
+
+1. In your Databricks workspace, go to **Settings** > **Developer**. Next to **Access tokens**, click **Manage**.
+1. Click **Generate new token**, enter a comment, and click **Generate**. Take note of your token.
+1. In Datadog, open the Databricks integration tile.
+1. On the **Configure** tab, click **Add New**. 
+1. Enter a workspace name, your Databricks workspace URL, and the Databricks token you generated.
+   {{< img src="data_jobs/databricks/configure-token.png" alt="In the Datadog-Databricks integration tile, a Databricks workspace is displayed. This workspace has a name, URL, and API token." style="width:100%;" >}}
+
+
+### Install the Datadog Agent on your Databricks cluster(s)
+
+Use [this init script][2] to install the Datadog Agent globally, or on a specific Databricks cluster.
```

Comment:
Can we give the customer this script as the deafault one that collects logs by default: https://drive.google.com/file/d/19Z50fVzeoWbHU4L4XwPJspGmOyx_UX3R/view?usp=drive_link that does include 

For the script you are currently linking to, we can show that one as an option if the customer doesn't want logs collected.